policyengine_canada/variables/gov/provinces/mb/tax/income/credits/tuition_amount/mb_tuition_amount_credit.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_canada.model_api import *
+
+
+class mb_tuition_amount_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "Manitoba tuition amount credit"
+    definition_period = YEAR
+    defined_for = "mb_tuition_amount_credit_eligible"
+
+    def formula(person, period, parameters):
+        p = parameters(
+            period
+        ).gov.provinces.mb.tax.income.credits.tuition_amount
+
+        tuition = person("tuition_expenses", period)
+
+        # check if full-time student
+        full_time = person("is_full_time_student", period)
+
+        # check if disabled
+        disabled = person("is_disabled", period)
+
+        tuition_addition = select(
```

Comment:
They're evaluated in sequence, and `default` speeds up `select` statements.
```suggestion
        tuition_addition = select(
            [full_time, disabled],
            [p.amount.full_time, p.amount.part_time.disabled],
            default=p.part_time.non_disabled
        )
```